core/src/io/anuke/mindustry/input/InputHandler.java

```diff
@@ -820,6 +828,8 @@ void iterateLine(int startX, int startY, int endX, int endY, Cons<PlaceLine> con
 
         if(diagonal){
             points = Placement.pathfindLine(block != null && block.conveyorPlacement, startX, startY, endX, endY);
+        }else if(follower){
+            points = Placement.followLineUntil(startX, startY, endX, endY);
```

Comment:
What happens if you try to upgrade walls or some other non-conveyor non-conduit like this?

Reply:
oh, then it just follows that line till the end, but it won't go around corners, so not that useful